src/main/java/school/lemon/changerequest/java/vehicles/Airplane.java

```diff
@@ -0,0 +1,92 @@
+package school.lemon.changerequest.java.vehicles;
+
+/**
+ * Created by Yaroslav Pavlinskiy on 04.12.2016.
+ */
+public class Airplane implements IAirplane {
+    private int manufacturedYear;
+    private String make;
+    private String model;
+    private int maximumHeightFeet;
+
+    public Airplane(int manufacturedYear, String make, String model, int maximumHeightFeet) {
+        this.manufacturedYear = manufacturedYear;
+        this.make = make;
+        this.model = model;
+        this.maximumHeightFeet = maximumHeightFeet;
+    }
```

Comment:
You have got a lot of duplicated code: `manufacturedYear`, `make` and `model` as parameters of Airplane **AND** Car **AND** Boat, so it'll be better to move them into some abstract predecessor.

I'll not duplicate this comment, but it also applies to Bot and Car implementations.